app/BlinkLab/BlinkLab.py

```diff
@@ -133,74 +141,91 @@ def run(self):
                 
+                # call 
+                self.mgr.dn_setACLEntry(
+                    macAddress   = m,
+                    joinKey      = [ord(b) for b in 'DUSTNETWORKSROCK'],
+                    
+                )
+            
+            # reset the network (reset sytstem: type = 0, reset specific motes: type= 2)
+            print 'self.mgr.dn_reset'
+            self.mgr.dn_reset(
+                type       = 0,
+                macAddress = [0x00]*8,
             )
-        
-        # reset the network (reset sytstem: type = 0, reset specific motes: type= 2)
-        print 'self.mgr.dn_reset'
-        self.mgr.dn_reset(
-            type       = 0,
-            macAddress = [0x00]*8,
-        )
-        print 'self.mgr.disconnect()'
-        self.mgr.disconnect()
-        time.sleep(30)
-        print 'self.mgr.connect()'
-        self.mgr.connect({'port': SERIALPORT_MGR})
-        
-        self.mgrsub = IpMgrSubscribe.IpMgrSubscribe(self.mgr)
-        self.mgrsub.start()
-        self.mgrsub.subscribe(
-            notifTypes =    IpMgrSubscribe.IpMgrSubscribe.ALLNOTIF,
-            fun =           handle_mgr_notif,
-            isRlbl =        False,
-        )
-        
-        # wait for network to form
-        while True:
+            print 'self.mgr.disconnect()'
+            self.mgr.disconnect()
+            time.sleep(30)
+            print 'self.mgr.connect()'
+            self.mgr.connect({'port': SERIALPORT_MGR})
             
-            res = self.mgr.dn_getNetworkInfo()
-            print 'size of network as parameter: {0}, res.numMotes={1}, {2}'.format(networksize,res.numMotes, res)
-            if res.numMotes==networksize:
-                break
-            time.sleep(1)
-        
-        # blink transactions
-        for t in range(1):
+            self.mgrsub = IpMgrSubscribe.IpMgrSubscribe(self.mgr)
+            self.mgrsub.start()
+            self.mgrsub.subscribe(
+                notifTypes =    IpMgrSubscribe.IpMgrSubscribe.ALLNOTIF,
+                fun =           handle_mgr_notif,
+                isRlbl =        False,
+            )
             
-            # blink packets
-            for p in range(1):
-                print 'send packet {0} of transaction {1}'.format(p,t)
-                self.tag.dn_blink(
-                    fIncludeDscvNbrs = 1,
-                    payload          = [t,p],
-                )
-                time.sleep(5)
+            # wait for network to form
+            while True:
                 
-            # reset tag
-            print 'self.tag.dn_reset()'
-            self.tag.dn_reset()
-
+                res = self.mgr.dn_getNetworkInfo()
+                print 'size of network as parameter: {0}, res.numMotes={1}, {2}'.format(networksize,res.numMotes, res)
+                if res.numMotes==networksize:
+                    break
+                time.sleep(1)
+            
+            blinkPayLoad = [00,11,22,33]
```

Comment:
we want the payload of each blink packet to contain the transaction number (`t` below) and the packet number withing that transaction (`p` below). I don't understand why you are defining a variable here for a static payload, and removing the `t,p` contents we had before. Please revert.